src/BuildASub.java

```diff
@@ -60,4 +69,60 @@ public static String chooseMeat() throws Exception {
             throw new Exception("Meat not set!");
         }
     }
+
+    public static Boolean addSalad (){
+        Boolean subSalad = false;
+
+        Scanner scan = new Scanner(System.in);
+        System.out.println("Do you want salad?:\n" +
+        "\t1. Yes\n" +
+        "\t2. No\n");
+        int saladChoice = scan.nextInt();
+        if(saladChoice == 1){
```

Comment:
Just a suggestion here, the cheese and toast branch read in the Y or N character. While it is less robust it might make it a little more consistent